README.md

```diff
@@ -149,6 +149,7 @@ If you're using the default tenant, the URL is `ws://realtime-dev.localhost:4000
 | TENANT_MAX_JOINS_PER_SECOND          | string  | The default value of maximum channel joins per second that each tenant can support, used when creating a tenant for the first time. Defaults to '100'.                                                                                                                                                                          |
 | SEED_SELF_HOST                       | boolean | Seeds the system with default tenant                                                                                                                                                                                                                                                                                            |
 | SELF_HOST_TENANT_NAME                | string  | Tenant reference to be used for self host. Do keep in mind to use a URL compatible name                                                                                                                                                                                                                                         |
+| LOG_LEVEL                            | string  | Sets log level for Realtime logs. (default: info)
```

Comment:
I think it would be helpful to enumerate all possible options explicitly, given that you're using to_existing_atom, which requires predefined atoms

Reply:
will add